.github/workflows/tests.yml

```
@@ -0,0 +1,30 @@
+# Authored by Alexandra N. Walker <alex.walker@indicio.tech> ☀️
+
+name: Tests
+on:
+  push:
+    branches: 
+      - main
+      - feature/revocation
+      - test/tests
+  pull_request:
+    branches:
+      - main
+      - feature/revocation
```

Probably a good idea to make this run on all feature branches :slightly_smiling_face: 

```suggestion
      - feature/**
```